third_party/move/move-compiler-v2/src/pipeline/reference_safety_processor.rs

```diff
@@ -1411,6 +1470,11 @@ impl<'env, 'state> LifetimeAnalysisStep<'env, 'state> {
     ) {
         let label = self.state.make_temp(src, self.code_offset, 0, false);
         let child = self.state.replace_ref(dest, self.code_offset, 1);
+        self.state
```

Comment:
Code might be clearer if you made a helper function for this with a useful name.  You have the same 5-line expression at line 1524.

Reply:
Good point, done.